tangos/properties/yt/basic.py

```diff
@@ -0,0 +1,65 @@
+from .. import PropertyCalculation, LivePropertyCalculation
+import numpy as np
+import tangos as db
+
+class FindCenter(PropertyCalculation):
+    """Returns center array in halo finder units"""
+    names = "Center"
```

Comment:
Ideally names of properties implemented in tangos don't have a capital letter in them unless there is a particular reason. e.g. `M200` deserves a capital because it is referring to a variable name with the capital, but `Center` to my mind doesn't because it's just a normal word (so I'd call it `center`). I realise there are a few exceptions, but generally this is true. Is it possible to make the property names lower case here, or will that break your workflow?

Reply:
That's fine! I've changed the variable names so that most are lower case. The only ones I left as is were those that are translated directly out of rockstar (e.g., X_Mpc). Is this okay? Or would you prefer that these be lower case, as well?